src/components/StoriesList/StoriesList.module.scss

```diff
@@ -1,82 +1,130 @@
 @import "../../styles/variables";
 
 .stories {
-  background: $grey;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding-bottom: 50px;
+  padding-bottom: 30px;
+  margin-top: 50px;
+  background: lighten($grey, 10%);
```

Comment:
On my screen this reads as barely there. To the point where it seems like a half-hearted afterthought rather than a deliberate decision to add contrast between the sections. Maybe it looks better on nicer screens but I think it should be a bit darker. I do think $grey is too dark, but maybe split the difference?